README.md

```diff
@@ -69,6 +69,187 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 See code for all available configurations.
 
+<<<<<<< HEAD
```

Comment:
Please fix the merge conflict.

Reply:
Merge conflicts should be corrected now.